src/android/notification/Builder.java

```diff
@@ -327,11 +327,13 @@ private void applyDeleteReceiver(NotificationCompat.Builder builder) {
 
         PendingIntent deleteIntent;
 
-        if(SDK_INT >= 31){
+        if (SDK_INT >= 34) {
```

Comment:
What do you think about to have a local variable that based in the SDK version have PendingIntent.FLAG_IMMUTABLE or 33554432 and that variable is what is passed to  deleteIntent = PendingIntent.getBroadcast(
                    context, reqCode, intent, FLAG_UPDATE_CURRENT | 33554432);

Reply:
Yes, I actually considered that. We can do that. By the way, this PR is still in draft, it is not ready for review.

Reply:
That won't work for the else case though. On that case, we're not passing anything in the mutability flag.

Reply:
I was actually able to refactor and implement your suggestion. Thank you!

Reply:
Since I applied the requested changes, I am going to resolve this.